game-of-agents.py

```diff
@@ -0,0 +1,83 @@
+﻿from envs import *
+from agents import *
+
+
+def inrow_heuristic(env, player, weight=0):
+    """
+    Score system:
+        win = inf
+        loss = -inf
+        array of some symbols without space to extend = 0
+        array with 1 symbol = +/-1 = +/-(1*1)
+        array with 2 symbols = +/-4 = +/-(2*2)
+        array with 3 symbols = +/-9 = +/-(3*3)
+    This scores growth if the array have multiply space to extend to!!!
+    """
+    # Check terminal state
+    if env.is_terminal_state():
+        if env.available_moves(player) > 0:
```

Comment:
`env.available_moves` always returns an iterable, not a number... rethink your logic.

Reply:
fixed